seaworthy/dockerhelper.py

```diff
@@ -116,23 +155,41 @@ def create_container(self, name, image, network=None, **kwargs):
             namespace.
         :param image:
             The image tag or image object to create the container from.
-        :param docker.models.networks.Network network:
+        :param network:
             The network to connect the container to. The container will be
             given an alias with the ``name`` parameter. Note that, unlike the
-            Docker Python client, this parameter should be a ``Network`` model
-            object, and not just a network ID.
+            Docker Python client, this parameter can be a ``Network`` model
+            object, and not just a network ID or name.
+        :param volumes:
+            A mapping of volumes to bind parameters. The keys to this mapping
+            can be any of three types of objects:
+            - A ``Volume`` model object
+            - The name of a volume (str)
+            - A path on the host to bind mount into the container (str)
```

Comment:
Should we support [`PathLike`](https://docs.python.org/3/library/os.html#os.PathLike) objects?

Reply:
Sure, if it's minimal extra effort.

Reply:
Oh, wait. As a dict key? I guess we probably shouldn't in that case.